commit

```diff
@@ -1,26 +1,36 @@
 #!/usr/bin/env bash
+set -euo pipefail
 
-echo 'Running commit script'
-
-IGNORED_BRANCHES=("dev" "master" "qa" "uat" "staging")	
+IS_IGNORED=false
 IGNORED_PATH="./.ignore"
+CURRENT_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+IGNORED_BRANCHES=("dev" "master" "qa" "uat" "staging")
+
+colorEcho(){
+    RED="\033[0;31m"
+    GREEN='\033[0;32m'
+    YELLOW='\033[1;33m'
+    NO_COLOR='\033[0m'
+    printf "${!1}${2} ${NO_COLOR}\n"
+}
 
-if [ -f "$IGNORED_PATH" ]
+commitWithBranch() {
+    colorEcho "GREEN" "Commit translation: \"$CURRENT_BRANCH: $1\""
+    # Edit your config here
+    git commit -m "$CURRENT_BRANCH: $1"
+}
+
+if [[ -f "$IGNORED_PATH" ]]
 then
     CUSTOM_BRANCHES=$(cat "$IGNORED_PATH")
     BRANCHES=( $CUSTOM_BRANCHES )
     IGNORED_BRANCHES=( ${IGNORED_BRANCHES[@]} ${BRANCHES[@]} )
-fi	
-
-CURRENT_BRANCH_CMD="git rev-parse --abbrev-ref HEAD"
-CURRENT_BRANCH=$(eval $CURRENT_BRANCH_CMD)
-
-COMMIT_WITH_BRANCH="git commit -m \"$CURRENT_BRANCH: $1\""
-DEFAULT_COMMIT="git commit -m \"$1\""
+    colorEcho "YELLOW" "Excluding branches from .ignore file."
```

Comment:
Not sure if we need any of this information.

Reply:
Since there’re confusing conditions like default ignoring branches and branches to ignore from .ignore file, I thought user may want to know what commit was really made.

Reply:
The use will know from the commit message in the end.

Reply:
Yes but making it fancy 🤣